tardis/montecarlo/base.py

```diff
@@ -134,10 +135,14 @@ def __init__(
         self.virt_packet_initial_rs = np.ones(2) * -1.0
         self.virt_packet_initial_mus = np.ones(2) * -1.0
 
+        self.r_packet_tracking = np.zeros(1)
```

Comment:
Should this be a boolean?

Reply:
I don't think so this needs to be a boolean. This property of the runner actually stores the `r_packet_track` array's data. Thus it is just initialised in this place.